src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -87,9 +87,13 @@ function ($deferred, $namespace, &$expiredIds) use ($getId) {
      * @param LoggerInterface|null $logger
      *
      * @return AdapterInterface
+     *
+     * @deprecated since Symfony 4.2.
```

Comment:
no dot at the end (I know, our conventions are hard to follow :)).

Reply:
updated